packages/commonjs/package.json

```diff
@@ -46,7 +46,7 @@
     "require"
   ],
   "peerDependencies": {
-    "rollup": "^1.20.0||^2.0.0"
+    "rollup": "^1.20.0||^2.3.4"
```

Comment:
I'm good with this, but it will technically be a breaking change, so please make sure to indicate that in the PR description

Reply:
I've ticked the "breaking change" box, do you expect anything else?

Reply:
Actually I think this needs to be 2.3.4 only, it won't work on 1.20.0 anymore

Reply:
A brief description in your own words of what the breaking change is would be a helpful reminder when we get to the point it's ready for merge :)

Reply:
I updated the PR description.